10-angular/introduction_to_angular/readme.md

```diff
@@ -71,20 +71,11 @@ Files/folders:
 $ mkdir todo_angular
 $ cd todo_angular
 $ touch index.html
-$ mkdir js
-$ mkdir js/controllers
-$ touch js/app.js
+$ mkdir app
+$ mkdir app/controllers
+$ touch app/app.js
 ```
```

Comment:
I think if we are trying to stay in line with the conventions the rest of the week we want something like `js/todos` as a file structure, no `controllers` directory, rather following `index.controller.js` conventions